src/signup/signup.hellpers.js

```diff
@@ -0,0 +1,24 @@
+const fsPromises = require('fs').promises;
```

Comment:
I guess, it would be better to call that file `signup.model`.

Reply:
done